tests/sitemaps_tests/test_generic.py

```diff
@@ -45,3 +45,16 @@ def test_generic_sitemap(self):
 </urlset>
 """ % expected
         self.assertXMLEqual(response.content.decode(), expected_content)
+
+    def test_generic_sitemap_lastmod(self):
+        test_model = TestModel.objects.first()
```

Comment:
I can submit a commit with this suggestion instead of `TestModel.objects.first()`